### ISSUE TITLE
about graph classfication

### ISSUE THREAD

**Issue by GitHubSOLUTE** [meliketoy/graph-cnn.pytorch#3]
What can I do to achieve graph classfication? I know it can be done to add pooling layer and softmax, and change loss function, etc. 
The question is, now the model's input is like one graph, I have no idea that whether I need to change the model' input or architecture.
